archive/data/music.json

```diff
@@ -20271,9 +20271,8 @@
     "referencedBy": [],
     "date": "2012-04-02T01:00:00.000Z",
     "directory": "temporal-shenanigans",
-    "bandcampId": 1885000843,
+    "bandcampId": null,
```

Comment:
okay well now you're just doing the opposite of the thing

Reply:
destiny's path twists and turns, we can but walk down it